controllers/statefulset.go

```diff
@@ -0,0 +1,316 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package controllers
+
+import (
+	"context"
+	"reflect"
+	"time"
+
+	searchv1alpha1 "github.com/stolostron/search-operator/api/v1alpha1"
+	appv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	redisconfigmap = "search_redisgraph_customization"
```

Comment:
The name `search-redis-conf` is more consistent.

- we use `-` instead of `_` for most names.
- `redis-conf` matches `redis.conf`, which is the default name used by redis docs.

Reply:
Updated to `redisconfigmap = "redis-conf"`

Reply:
and `redisConfigmap`